src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -496,7 +499,41 @@ TEST_CASE(
     "correctly")
 {
   PerfAnalyzerParameters params{};
-  TestRequestRateManager trrm{params};
+  bool is_sequence_model{false};
+  bool async{false};
+  bool streaming{false};
+
+  const auto& ParameterizeAsyncAndStreaming{[](bool& async, bool& streaming) {
+    SUBCASE("sync non-streaming")
+    {
+      async = false;
+      streaming = false;
+    }
+    SUBCASE("async non-streaming")
+    {
+      async = true;
```

Comment:
are async and streaming ever actually set in params?

Reply:
Good catch, fixed that and ran locally to confirm unit test passes. Still started another CI pipeline.

Reply:
I think you missed my main review comment:

_If this is working and passing, please apply this into the side branch instead of main. It is a small change and should be easy to apply. If not, I think I will have to rebase everything again._

Reply:
I rebased onto the side branch without updating the PR target. Target is now updated.